src/index.css

```diff
@@ -62,41 +40,55 @@ ul {
 }
 
 .book-category {
-  color: #121212;
-  font-family: 'Montserrat', sans-serif;
-  font-weight: bold;
+  font-weight: bolder;
+  opacity: 0.6;
+  font-size: 0.875rem;
```

Comment:
- [ ] Nice job implemented CSS best practices, however, there are a few repeating rules that needs to be addressed,
`font-size: 0.875em` had been repeated, we can easily create a separate class for it and apply it on the HTML element